dev-python/pew/pew-1.1.1.ebuild

```diff
@@ -23,4 +23,5 @@ DEPEND="${RDEPEND}
 	>=dev-python/setuptools-17.1[${PYTHON_USEDEP}]
 	>=dev-python/virtualenv-1.11.6[${PYTHON_USEDEP}]
 	>=dev-python/virtualenv-clone-0.2.5[${PYTHON_USEDEP}]
-	>=dev-python/pythonz-bd-1.11.2[${PYTHON_USEDEP}]"
+	>=dev-python/pythonz-bd-1.11.2[${PYTHON_USEDEP}]
+	>=dev-python/shutilwhich-1.1.0[${PYTHON_USEDEP}]"
```

Comment:
Please sort the deps. I get someone failed at it already but please don't make it worse.